dags/analyzer_helper/tests/integration/test_discord_is_user_bot.py

```diff
@@ -0,0 +1,102 @@
+import unittest
+from datetime import datetime
+
+from analyzer_helper.discord.utils.is_user_bot import UserBotChecker
+from bson import ObjectId
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class TestUserBotChecker(unittest.TestCase):
+    def setUp(self):
+        self.client = MongoSingleton.get_instance().client
+        self.platform_id = "discord"
+        self.db = self.client[self.platform_id]
+        self.rawinfo_collection = self.db["rawinfos"]
+        self.guildmembers_collection = self.db["guildmembers"]
```

Comment:
`rawinfos` and `guildmemebrs` collections are under the `guildId` database and not the `platform_id` database.